pyabc/storage/history.py

```diff
@@ -758,21 +816,26 @@ def get_population_strategy(self):
         return json.loads(abc.population_strategy)
 
     @with_session
-    def get_population_extended(self, *, m=None, t="last", tidy=True) \
+    def get_population_extended(self, *, m: Union[int, None] = None,
+                                t: Union[int, str] = "last",
+                                tidy: bool = True) \
             -> pd.DataFrame:
         """
         Get extended population information, including parameters, distances,
         summary statistics, weights and more.
 
         Parameters
         ----------
-        m: int, optional
-            The model to query.
-            If omitted, all models are returned
-        t: str, optional
-            Can be "last" or "all"
+
+        m: int or None, optional (default = None)
+            The model to query. If omitted, all models are returned
+
+        t: int or str, optional (default = "last")
+            Can be "last" or "all", or a population index.
             In case of "all", all populations are returned.
-            If "last", only the last population is returned.
+            If "last", only the last population is returned, for an int value
+            only the corresponding population at that time index.
+
```

Comment:
too picky: but the case of `population index` given is not explained :dancing_women: